client/src/components/StepWizard/WizardFormNav.js

```diff
@@ -26,7 +25,7 @@ export const StyledButtonWizard = styled(StepWizard)`
       min-height: 100%;
     }
   }
-
+  
```

Comment:
remove the 2 white spaces on line 28 - do not remove the line, just the white spaces